source/Hooks.cpp

```diff
@@ -26,11 +26,11 @@ namespace hooks
 			RE::TESQuest* quest;
 			RE::BGSInstancedQuestObjective objective;
 			RE::TESQuestTarget* target;
-			for (int i = 0; i < player->objectives.size(); i++) {
-				for (int j = 0; j < player->objectives[i].objective->numTargets; j++) {
-					if (a_questTarget->unk00 == (uint64_t)player->objectives[i].objective->targets[j]) {
-						quest = player->objectives[i].objective->ownerQuest;
-						objective = player->objectives[i];
+			for (int i = 0; i < player->GetPlayerRuntimeData().objectives.size(); i++) {
+				for (int j = 0; j < player->GetPlayerRuntimeData().objectives[i].objective->numTargets; j++) {
+					if (a_questTarget->unk00 == (uint64_t)player->GetPlayerRuntimeData().objectives[i].objective->targets[j]) {
+						quest = player->GetPlayerRuntimeData().objectives[i].objective->ownerQuest;
+						objective = player->GetPlayerRuntimeData().objectives[i];
```

Comment:
Note that the 1.6.629+ update for the rest of the code hasn't been committed to the repository, so you might want to revert these specific changes and then commit a full pass 1.6.629+ update after merge.